test/enum_set_test.cpp

```diff
@@ -20,75 +20,123 @@
 
 namespace {
 
+using libspirv::EnumSet;
 using libspirv::CapabilitySet;
 using spvtest::ElementsIn;
 using ::testing::Eq;
 using ::testing::ValuesIn;
 
-TEST(CapabilitySet, DefaultIsEmpty) {
-  CapabilitySet c;
+TEST(EnumSet, IsEmpty1) {
+  EnumSet<uint32_t> set;
+  EXPECT_TRUE(set.IsEmpty());
+  set.Add(0);
+  EXPECT_FALSE(set.IsEmpty());
+}
+
+TEST(EnumSet, IsEmpty2) {
+  EnumSet<uint32_t> set;
+  EXPECT_TRUE(set.IsEmpty());
+  set.Add(150);
+  EXPECT_FALSE(set.IsEmpty());
+}
+
+TEST(EnumSet, IsEmpty3) {
+  EnumSet<uint32_t> set(4);
+  EXPECT_FALSE(set.IsEmpty());
+}
+
+TEST(EnumSet, IsEmpty4) {
+  EnumSet<uint32_t> set(300);
+  EXPECT_FALSE(set.IsEmpty());
+}
+
+TEST(EnumSet, HasAnyOf) {
```

Comment:
You should split this into multiple tests, so each behaviour is tested separately.
Ideally, each test should have the structure:  arrange, act, assert.
Also, the guidance on naming is that test names should include the condition being tested and the expected outcome.

Why?  
- separation of concerns.  if something fails some day, it should be clear what went wrong and why.  It should be very isolated
- the name of the test should clearly indicate the intent.  That helps in diagnosing and responding to failures, or when we have to rewrite tests due to other changes in the code being tested.